.github/workflows/check-cherry-pick.yml

```diff
@@ -0,0 +1,76 @@
+name: "Check cherry-picks"
+on:
+  pull_request_target:
+
+permissions: {}
+
+env:
+  PICKABLE_BRANCHES: master staging
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'NixOS'
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Check cherry-picks
+      run: |
+        # Find alleged cherry-picks
```

Comment:
I think we should put that as script into the workflow directory to have editor support for highlighting and linting tools.

Reply:
I would agree except it emits github action-specific output (folded sections, warning & error lines)

Reply:
Actually I can just conditionally emit those lines if I detect we're running as a github action can't I - so yeah I might do this...